src/styles/base/_typography.scss

```diff
@@ -1,5 +1,5 @@
 // Robot Condensed:
-@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700);
```

Comment:
Yeah, I'm working with Goffredo to make the typography more unified. I should be using max two typefaces. Either way, I'm planning to put together a style guide for the Gazelle to use in the future so that everything stays unified